start

```diff
@@ -147,17 +147,17 @@ function process_args() {
       ENABLE="$1"
       shift
       ;;
-    --enable-soroban-rpc)
-      ENABLE_SOROBAN_RPC=true
+    --enable-stellar-rpc)
+      ENABLE_STELLAR_RPC=true
       ;;
     --enable-soroban-diagnostic-events)
       ENABLE_SOROBAN_DIAGNOSTIC_EVENTS=true
       ;;
     --disable-soroban-diagnostic-events)
       DISABLE_SOROBAN_DIAGNOSTIC_EVENTS=true
       ;;
-    --enable-soroban-rpc-admin-endpoint)
-      ENABLE_SOROBAN_RPC_ADMIN_ENDPOINT=true
+    --enable-stellar-rpc-admin-endpoint)
+      ENABLE_STELLAR_RPC_ADMIN_ENDPOINT=true
```

Comment:
Can we lose the stellar prefix here? We don't use the stellar prefixes in options, see below for core.